webpack.config.js

```diff
@@ -67,7 +69,11 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
   plugins: [
     new WebpackAssetsManifest({
       entrypoints: true,
-      publicPath: true,
+      publicPath(filename, plugin) {
+        return filename.endsWith('.js')
+          ? url.resolve(plugin.compiler.options.output.publicPath, filename)
+          : filename;
+      },
```

Comment:
Without this, `WebpackAssetsManifest` will prepend the public path to every assets, so you'd end up with something like...

```json
{
  "entrypoints": {
    "document-capture": {
      "assets": {
        "js": [
          "/packs/js/32.js",
          "..."
        ],
        "gif": [
          "/packs/spinner.gif",
          "/packs/spinner@2x.gif"
        ]
      }
    }
  }
}
```

This is the behavior we expect for JavaScript files, since they're not part of the asset pipeline, but in the way we're using images here, we want the original filename, unprefixed.

The logic is intended to retain [the default behavior](https://github.com/webdeveric/webpack-assets-manifest/blob/206edd4d90cec9249d6617f1c48c9e79471c010e/src/WebpackAssetsManifest.js#L894-L899) for JavaScript files, but use the plain filename for everything else.

Reply:
Added an inline comment to this effect in [bed0542](https://github.com/18F/identity-idp/pull/6198/commits/bed0542e8c3e096f7ec4fe27a7434c8dbd7872da).